run/apop2john.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+# This software is Copyright (c) 2021 Mark Silinio <mark.silinio-at-gmail.com>,
+# and it is hereby released to the general public under the following terms:
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted.
+#
+# Utility to bruteforce APOP hash
```

Comment:
I think we need to change this comment and the similar one in `radius2john.py` to say something more reasonable like, "Extract and format POP3 APOP challenge/responses for password cracking with JtR" here and "Extract and format suitable RADIUS packets for shared secret cracking with JtR" there.